src/components/EventTypeFilters.vue

```diff
@@ -1,12 +1,22 @@
 <template>
-  <div class="event-type-filters">
-    <h5 class="event-filters-title">Type of event</h5>
-    <div v-for="(label, type) in eventTypes">
-      <input type="checkbox"
-        class="no-margin-checkbox"
-        :checked="selectedEventTypes.includes(type)" 
-        @change="updateSelectedEventTypes(type, $event)"
-      /> {{label}}
+  <div>
+    <h5 v-if="showTitle" class="filter-events-title">Type of event</h5>
+    <div 
+      class="filter-events-item" 
+      v-for="(label, type) in eventTypes"
+    >
+      <div class="filter-events-checkbox">
```

Comment:
So the purpose of the way this arranged is to allow custom styling of the checkboxes via a selector like:

```
input:checked + label
```
I believe we wouldn't be able to use this trick for CSS-only fancy checkboxes if the input were inside the label. Unless I'm missing something! LMK.